.github/workflows/test_s3_minio.yml

```diff
@@ -5,11 +5,17 @@ name: S3/Minio Test
 on:
   push:
     branches:
-      - main
+      - main  # keep this at all times
+      - is-it-using-s3  # test on feature branch
```

Comment:
Not required now there is a PR
```suggestion
```

Reply:
See the following line, `pull_request:` - this will run the workflow against any branches that have an open PR, including when they're pushed to.

Reply:
ow no we don't want that, man - we want this thing to run only on this branch when committing to it - otherwise Github will ban us :laughing: